settings/settings.go

```diff
@@ -0,0 +1,31 @@
+package settings
+
+import (
+	"os"
+	"sync"
+)
+
+type Settings map[string]string
+
+var settings Settings
+var once sync.Once
+
+func getSetting(settings Settings, key string, defaultValue string) {
+	if value, present := os.LookupEnv(key); present {
+		settings[key] = value
+	} else {
+		settings[key] = defaultValue
+	}
+}
+
+func GetSettings() Settings {
+	once.Do(func() {
+		settings = make(Settings)
+		getSetting(settings, "GO_LAUNCH_A_SURVEY_LISTEN_HOST", "0.0.0.0")
+		getSetting(settings, "GO_LAUNCH_A_SURVEY_LISTEN_PORT", "8000")
+		getSetting(settings, "SURVEY_RUNNER_URL", "http://localhost:5000")
```

Comment:
If you default this to empty it would really well at just routing to the same url. which is nice when you are behind the ALB, as you don't need to specify anything.